reading-notes-intro-computation-programming-python/chapter-one_getting-started/quiz.md

```diff
@@ -24,38 +33,56 @@ The cubic root is a function that :
 - return the side length.
 
 Write down the previous sentence another way, eg. in declarative knowledge.
+> The cubic root of x is a number y such as y * y * y = x
 
 Write down an algorithm to find the cubic root of any positive number, if it exists.
+> See [code](code/quizz-chapter1-exercice.py)
  
 Then, write down its execution on input 99.
+> 4.62608528137207  is close to the cubic root of  99.
 
 ## Computer type
 
 Which type of computer is a desk calculator ?
+> A fixed-program computer, it can execute only one program
+> [discussion online](https://www.quora.com/Are-calculators-considered-to-be-computers-If-not-then-why-do-they-have-similar-functions-to-computers)
+
 Is the [Pascaline](https://en.wikipedia.org/wiki/Pascaline) a computer ? If not, why ? 
+> The Pascaline calculates and stores results so it could be considered as a computer in its larger definition, but the program cannot be changed so it cannot be considered as a computer.
```

Comment:
Well, the Pascaline is a desk calculator, so of desk calculator are computer, so do the Pascaline.